resteasy-rxjava2/src/test/java/org/jboss/resteasy/rxjava2/RxTest.java

```diff
@@ -62,6 +67,7 @@ public void after()
    }
 
    @Test
+   @Ignore

```

Comment:
Maybe, some comment, why this is ignored, would be useful ...

Reply:
I've updated and re-enabled the tests in RxTest in pull request #1493.